frontend/doc-site/05__Annotate and Analyze Your Data/5_2__Self Host cellxgene.mdx

```diff
@@ -1,23 +1,23 @@
 # Self-Host CellxGene
 
-## Hosting CellxGene Desktop on the Web
+## Hosting CZ CELLxGENE Annotate on the Web
 
-Cellxgene desktop is intended to be used by researchers on their local machines. In cases where collaborators are not comfortable installing cellxgene, it is possible to host cellxgene, so your collaborators only need to open the URL you send them.
+CZ CELLxGENE Annotate is intended to be used by researchers on their local machines. In cases where collaborators are not comfortable installing CellxGene, it is possible to host CellxGene, so your collaborators only need to open the URL you send them.
```

Comment:
```suggestion
CZ CELLxGENE Annotate is intended to be used by researchers on their local machines. In cases where collaborators are not comfortable installing Annotate, it is possible to host Annotate, so your collaborators only need to open the URL you send them.
```
Annotate used to be simply called "cellxgene". In this and other cases below and elsewhere in the documentation, we should either refer to it as "CELLxGENE Annotate" or "Annotate". 

Referring to it as CELLxGENE will create confusion, since that's now the parent brand.